go.mod

```diff
@@ -20,16 +20,17 @@ require (
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
+	github.com/ulikunitz/xz v0.5.8 // indirect
 	github.com/zorkian/go-datadog-api v2.30.0+incompatible
 	go.uber.org/zap v1.15.0
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
-	k8s.io/api v0.20.2
-	k8s.io/apiextensions-apiserver v0.20.2
-	k8s.io/apimachinery v0.21.0
-	k8s.io/cli-runtime v0.20.2
-	k8s.io/client-go v0.20.2
+	k8s.io/api v0.20.11
+	k8s.io/apiextensions-apiserver v0.20.11
+	k8s.io/apimachinery v0.21.5
```

Comment:
This is the only one with v0.21.X.
I guess it's OK because it was already like that. Pointing it out just in case.

Reply:
Probably a mistake, should be aligned, good catch

Reply:
I don't rememenber why it is like that. but it is also like this in the `datadog-agent` repo

Reply:
done